LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2019 Simon Bärlocher
+Copyright (c) 2019 Arillso

```

Comment:
Could you add ITIGO?

Reply:
In arillso project all role copyrights are held by arillso.